commons-vfs2/src/main/java/org/apache/commons/vfs2/provider/local/LocalFileRandomAccessContent.java

```diff
@@ -45,7 +45,7 @@
                 @Override
                 public int read() throws IOException {
                     try {
-                        return raf.readByte();
+                        return raf.readByte() & 0xFF;
```

Comment:
Let's avoid repeated use of a magic number. Please refactor into a constant that reused in the code and tests. Same for -1.

Reply:
Sure.
Seems we need a place to store the constant `0xFF`. IMO we should have a class to store constants like `0xFF`, which seems is not presented in VFS. What about adding a `VFSConstants` class in `org.apache.commons.vfs2.util`?

Reply:
For now I would just add it as a constant in `RamFileRandomAccessContent`. The only other place I see it used, is once in an unrelated sandbox class.